internal/services/monitor/monitor_aad_diagnostic_setting_resource_test.go

```diff
@@ -53,7 +56,9 @@ func testAccMonitorAADDiagnosticSetting_eventhubDefault(t *testing.T) {
 	r := MonitorAADDiagnosticSettingResource{}
 	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
 		{
-			Config: r.eventhubDefault(data),
+			// Last error may cause the resource still exist. Need to delete it before a new test.
```

Comment:
What is meant by 
> Last error

Reply:
The acctest that previously failed, but the test resources are not deleted, since the resource type is not at the resource group level.

Reply:
Is there anymore info on why the previous acctest failed?

Reply:
No more details, but only from the error message of failed test cases